src/actions/SplitWalletModalActions.tsx

```diff
@@ -1,44 +0,0 @@
-import * as React from 'react'
-
-import { ButtonsModal } from '../components/modals/ButtonsModal'
-import { Airship, showError } from '../components/services/AirshipInstance'
-import { lstrings } from '../locales/strings'
-import { ThunkAction } from '../types/reduxTypes'
-import { getWalletName } from '../util/CurrencyWalletHelpers'
-
-export function showSplitWalletModal(walletId: string, pluginId: string): ThunkAction<Promise<void>> {
-  return async (dispatch, getState) => {
-    const state = getState()
-    const { account } = state.core
-    const { currencyConfig, currencyWallets } = account
-    const edgeWallet = currencyWallets[walletId]
-
-    let bodyText = lstrings.fragment_wallets_split_wallet_first_confirm_message_mobile
```

Comment:
Delete strings: `fragment_wallets_split_wallet_first_confirm_message_mobile`, `fragment_wallets_split_wallet_bch_to_bsv`, `fragment_wallets_split_wallet`, and `string_split`.

They seem to be unused, at least in my initial check. If they are used anywhere else, then obviously TypeScript will complain and they should be put back.